pages/pipelines/incoming_webhooks.md.erb

```diff
@@ -0,0 +1,25 @@
+# Incoming Webhooks
+
+Incoming webhooks are sent to Buildkite by source control providers ([GitHub](/docs/integrations/github), [GitLab](/docs/integrations/gitlab), [Bitbucket](/docs/integrations/bitbucket), etc.) to trigger builds. This page covers the most frequent questions that arise regarding the security of incoming webhooks in Buildkite.    
+
+### What kind of information on incoming webhooks is logged by Buildkite? 
+Buildkite only logs and temporarily stores the incoming webhook information as it was received, the relevant HTTP headers, and the remote IP for diagnostics purposes. This information is soon erased and cannot be used for audit.
+ 
+### How long does Buildkite store the information regarding incoming webhooks?
+Buildkite stores the information on recent incoming webhooks received per pipeline for approximately a week. It is done for troubleshooting purposes only — for example, to diagnose why a given hook didn’t trigger a build. This functionality is not exposed in the UI.
+ 
+### What security information is stored in the secret in a Buildkite URL? Does it only say that I — as a user — am talking to Buildkite, and not that the webhook is triggered from my GitHub organization (or any other GitHub entity)? 
```

Comment:
```suggestion
### What security information is stored in the secret in a Buildkite URL?
```

looks better? What do you think?

Reply:
I need to think about it a bit.

Reply:
I'm not totally happy with just cutting out the rest of the question here, but I cannot quite rephrase the long question at the moment either. If the muse does visit me regarding this entry, I'll submit an extended version as a fix.